consensus/istanbul/backend/api.go

```diff
@@ -214,11 +217,113 @@ func (api *APIExtension) GetCommitteeSize(number *rpc.BlockNumber) (int, error)
 	}
 }
 
+type ValidationResult struct {
+	BlockNumber              uint64              `json:"blockNumber"`
+	Round                    byte                `json:"round"`
+	Proposer                 common.Address      `json:"proposer"`
+	ProposerFromBlock        common.Address      `json:"proposerFromBlock"`
+	IsValidProposer          bool                `json:"isValidProposer"`
+	Committee                common.AddressSlice `json:"committee"`
+	CommitteeSealedFromBlock common.AddressSlice `json:"committeeSealedFromBlock"`
+	CommitteeFromBlock       common.AddressSlice `json:"committeeFromBlock"`
+	IsValidCommittee         bool                `json:"isValidCommittee"`
+	IsValidSeal              bool                `json:"isValidSeal"`
+}
+
+func (api *APIExtension) ValidateConsensusInfo(block *types.Block) (ValidationResult, error) {
```

Comment:
This validate code can check below.
- proposer from block is same with calculated proposer
- committe from block is same with calculated committe
- All sealed committe is in the calculated committe